client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -241,12 +287,19 @@ export default connect(
 			  } )
 			: undefined;
 
+		const planSlug = getSitePlanSlug( state, siteId );
+		const isPremium = !! planSlug && isPremiumPlan( planSlug );
+		const isProfessional = ! isPremiumPlan && !! planSlug && isBusinessPlan( planSlug );
```

Comment:
I appreciate all these speed improvements but I wonder if it's worth it now that the code is little harder to grok?

Should `isPaidPlan` from line 302 be moved up here so that all similar lines are together?

Reply:
Good call, I moved these together for consistency.